### PR TITLE
fix: giscus use current lang as default，and add missing `loading` params

### PR DESCRIPTION
**Pull request by xiaoxiyao**

### CODE REVIEW

File: layouts/partials/comments/provider/giscus.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
+{{ $lang := .Language.LanguageCode }}
+{{ if not $lang }}
+    {{ $lang = .Language.Lang }}
```
Comment (CaiJimmy) [inline layouts/partials/comments/provider/giscus.html:3]: According to https://gohugo.io/content-management/multilingual/, `.Language.Lang` doesn't seem to be a public variable, only `.Language.LanguageCode`. 

Then this change can be simplified as `data-lang="{{ default $.Language.LanguageCode .lang }}"` I think

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks for the PR and sorry for the late reply!

Can you please filter out the unrelated changes? Or create a new PR with only the specified changes.

File: layouts/partials/comments/provider/giscus.html (hunk 1 above)
Comment (xiaoxiyao) [reply layouts/partials/comments/provider/giscus.html:3]: Thank you for your review and feedback. I've created a new pull request (#1075) that only includes the changes about giscus。
And about the `.Language.Lang` variable, you can see https://gohugo.io/methods/page/language/

**Comment by xiaoxiyao** [comment]
Move to #1075

**Pull request closed without merging**